.github/scripts/bash/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+BASH_PATH=${BASH_PATH:-~/$TARGET/bash}
+BASH_BUILD_PATH=$BUILD_PATH/bash
+BASH_SOURCE_PATH=$SOURCE_PATH/bash
+
+PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/$TARGET/:$TOOLCHAIN_PATH/lib/gcc/$TARGET/$GCC_VERSION/:$PATH"
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$TOOLCHAIN_PATH/bin
+
+mkdir -p $BASH_BUILD_PATH
+cd $BASH_BUILD_PATH
+
+if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$BASH_BUILD_PATH/Makefile" ]]; then
+    echo "::group::Configure Bash"
+        rm -rf $BASH_BUILD_PATH/*
+
+        CFLAGS="-std=gnu89 -DNEED_EXTERN_PC" \
+        $BASH_SOURCE_PATH/configure \
```

Comment:
Please add `WSLENV="$WSLENV:PATH/p" \` before this line or to `config.sh`. This will ensure that the correct `cygwin1.dll` is used through WSL interop.

Reply:
The cross-compilation has been resolved and WSL is not required now.